library/IvozProvider/Mapper/Sql/FaxesInOut.php

```diff
@@ -31,9 +31,8 @@ protected function _save(\IvozProvider\Model\Raw\FaxesInOut $model,
         $statusHaschanged = $model->hasChange("status");
 
         if ($isOutgoingFax && $statusHaschanged && $isPending) {
-            $faxSenderJob = new \IvozProvider\Gearmand\Jobs\FaxSender();
-            $faxSenderJob->setFaxInOut($model);
-            $faxSenderJob->send();
+            $ari = new \Asterisk\ARI\Connector();
+            $ari->sendFaxfileRequest($model);
```

Comment:
I would prefer to throw the exception within the Connector. You can register the error in portals/application/configs/clear/errors. Yaml (Requires the page to be reloaded, currently error translations are done with javascript)